packages/netlify-cms-media-library-cloudinary/src/index.js

```diff
@@ -54,7 +54,7 @@ async function init({ options = {}, handleInsert } = {}) {
   const resolvedOptions = { ...defaultOptions, ...integrationOptions };
   const cloudinaryConfig = { ...defaultConfig, ...providedConfig, ...enforcedConfig };
   const cloudinaryBehaviorConfigKeys = ['default_transformations', 'max_files', 'multiple'];
-  const cloudinaryBehaviorConfig = pick(cloudinaryConfig, cloudinaryBehaviorConfigKeys);
+  const cloudinaryBehaviorConfig = pick(resolvedOptions, cloudinaryBehaviorConfigKeys);
```

Comment:
`resolvedOptions` is made up of keys that are internal to Netlify CMS, like `use_secure_url`. `cloudinaryConfig` has the value of `media_library.config` from the CMS configuration, which contains Cloudinary specific keys exclusively.

Reply:
`resolvedOptions` contains `defaultOptions` and `integrationOptions ` (line 54). It is used to create `cloudinaryBehaviorConfig` which contains Cloudinary related configurations defined under `cloudinaryBehaviorConfigKeys`. 

`cloudinaryConfig` contains `api_key` and `cloud_name` and only needs to be passed when creating the media library. The `mediaLibrary.show` function  requires `default_transformations` which is now passed through `resolved_options`. Without this, the default transformations were not applied to the inserted image.

Reply:
> `cloudinaryConfig` contains `api_key` and `cloud_name` and only needs to be passed when creating the media library.

`cloudinaryConfig` contains all configuration for the cloudinary media player. This line of code isn't passing `cloudinaryConfig`, it's using `lodash.pick` to select the behavior config keys from it.

Reply:
This is a screenshot from the console while running the branch from this PR. Notice the keys present in `resolvedConfig` - they're keys invented for the Netlify CMS integration, not anything that the Cloudinary widget would be able to use.

<img width="233" alt="screen shot 2019-02-06 at 10 22 47 am" src="https://user-images.githubusercontent.com/2112202/52351695-44d64780-29f9-11e9-9ad7-f01acf4b7339.png">

Reply:
Did you have the `default_transformation` defined in the config? Here's what I have for `resolvedOptions`
<img width="533" alt="screen shot 2019-02-08 at 4 15 29 pm" src="https://user-images.githubusercontent.com/4754103/52513710-6d10a280-2bc1-11e9-96f4-47b8008dee2d.png">